### DISCUSSION TITLE
Does mxnet work with GTX 1050?

### DISCUSSION THREAD

**Discussion by johnbroughton2017** [apache/incubator-mxnet#12461] [Q&A]
Hi,

I am having problem with running mxnet on my GTX 1050 + Ubuntu 16.04. The following error showed up when running a prediction with a pre-trained model. 

Have anyone used mxnet on GTX 1050? Any feedback is highly appreciated. 

Thanks.
- J

> [11:16:18] src/resource.cc:262: Ignore CUDA Error [11:16:18] src/storage/./pooled_storage_manager.h:77: CUDA: invalid device pointer
> 
> Stack trace returned 10 entries:
> [bt] (0) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x31f81a) [0x7fee2aa6c81a]
> [bt] (1) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x31fe41) [0x7fee2aa6ce41]
> [bt] (2) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x29ebeec) [0x7fee2d138eec]
> [bt] (3) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x29ed4c1) [0x7fee2d13a4c1]
> [bt] (4) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x29f2c87) [0x7fee2d13fc87]
> [bt] (5) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x29f49ec) [0x7fee2d1419ec]
> [bt] (6) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x2480f3d) [0x7fee2cbcdf3d]
> [bt] (7) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x248abf4) [0x7fee2cbd7bf4]
> [bt] (8) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x2494ad1) [0x7fee2cbe1ad1]
> [bt] (9) /usr/lib/x86_64-linux-gnu/libmxnet.so(+0x2487540) [0x7fee2cbd4540]

**Comment by vrakesh** [comment]
@johnbroughton2017 Surprised it is having an issue on ubuntu and a pascal card, I have tried on a 1060 and ubuntu, it works fine. Could you verify if CUDA is installed correctly, to verify further, could you try a nvidia docker based implementation?

Also requesting to add a more detailed reproducible example.

@mxnet-label-bot [CUDA, Question]

**Comment by haojin2** [comment]
Can you share your system info such as CUDA version/driver version? Please use tools/diagnose.py and paste the outputs of it here. I've previously used Ubuntu16.04+CUDA8+CuDNN6+GTX1050 and it worked fine on my end.

**Comment by vrakesh** [comment]
@johnbroughton2017 Any success with the installation?? were you able to get it running

**Comment by johnbroughton2017** [comment]
@vrakesh @haojin2 Hey guys, really appreciate your feedback.

I was using Ubuntu 16.04+CUDA9+CUDNN7+GTX1050. 
The same setup seems to work well with my 1080 card but fails with 1050. 
I also have tried on 1060 and it works well. 

It's a bit strange to me as I thought all PASCAL card should be supported in the same way.

-- J

PS: Trying the tools/diagnose.py now ...

**Comment by mariussoutier** [comment]
18.04, CUDA9.0, CUDNN7, GTX1050 (mobile), NVidia driver 396 works for me. Actually it's super fast.

**Comment by piyushghai** [comment]
@johnbroughton2017 Were you able to get it running using the configuration described above ?
